cluster/aws/util.sh

```diff
@@ -816,6 +816,7 @@ function kube-up {
     echo "readonly LOGGING_DESTINATION='${LOGGING_DESTINATION:-}'"
     echo "readonly ELASTICSEARCH_LOGGING_REPLICAS='${ELASTICSEARCH_LOGGING_REPLICAS:-}'"
     echo "readonly ENABLE_CLUSTER_DNS='${ENABLE_CLUSTER_DNS:-false}'"
+    echo "readonly ENABLE_CLUSTER_UI='${ENABLE_CLUSTER_UI:-false}'"
```

Comment:
Should be enabled by default for all providers. In this particular the logic is to take value from  cluster/aws/config-{default, test}.sh if specified. Otherwise take false as a default. Since there is ENABLED_CLUSTER_UI defined in both config files here the value would be equal to them (true by default).